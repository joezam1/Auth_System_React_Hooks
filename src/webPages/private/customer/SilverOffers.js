import React from "react";
import LinkButtonPrivateRedirect from "../../../components/LinkButtonPrivateRedirect";
import RouteConfig from '../../../../configuration/routes/RouteConfig.js';

export default function SilverOffers(){

    return(<div className="customer-silver-offers-section">
        This is the silver offers section
        <br />
        <h3>Go To</h3>

        <LinkButtonPrivateRedirect redirectToLocation={RouteConfig.privateCustomerDashboardPath} buttonText=" Customer Dashboard " />
        <br />
    </div>)
}