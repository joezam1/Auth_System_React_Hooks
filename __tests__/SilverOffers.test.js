import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import SilverOffers from '../src/webPages/private/customer/Silveroffers.js';


describe('File: SilverOffers.js', function(){
    afterEach(cleanup);
    describe('Component SilverOffers', function(){
        let container;
        beforeEach(()=>{
            container = render( <MemoryRouter> <SilverOffers/>  </MemoryRouter>  );
        });

        test('It should take a snapshot', function(){
            expect(container.asFragment(<SilverOffers/>)).toMatchSnapshot();
        });


    });
});