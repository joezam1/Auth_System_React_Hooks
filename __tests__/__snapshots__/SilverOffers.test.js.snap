// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File: SilverOffers.js Component SilverOffers It should take a snapshot 1`] = `
<DocumentFragment>
   
  <div
    class="customer-silver-offers-section webpage hide"
  >
    <div
      class="outerLayout"
    >
      <div
        class="header-container"
      >
        <div
          class="header-title"
        >
          <span>
              Hi 
            <strong>
               
            </strong>
          </span>
        </div>
        <div
          class="header-title"
        >
          <span>
             Status: 
            <strong>
                
            </strong>
             
          </span>
        </div>
      </div>
      <div
        class="container"
      >
        <h2>
          SILVER OFFERS Section.
        </h2>
        <div
          class="top-navigation-bar"
        >
          <h3>
            Go To
          </h3>
          <div
            class="button-private-redirect-section hide"
          >
            <button
              class="btnRedirect"
              data-testid="button-private-redirect-id"
              type="button"
            >
                Customer Dashboard  
            </button>
          </div>
        </div>
      </div>
      <div
        class="flex-container"
      >
        <div
          class="product"
        >
          item1
        </div>
        <div
          class="product"
        >
          item2
        </div>
        <div
          class="product"
        >
          item3
        </div>
        <div
          class="product"
        >
          item4
        </div>
        <div
          class="product"
        >
          item5
        </div>
        <div
          class="product"
        >
          item6
        </div>
      </div>
      <div
        class="container hide"
      >
        <span>
          Page 1 
        </span>
      </div>
    </div>
  </div>
    
</DocumentFragment>
`;
